apps/datahub/src/app/home/search/organization-preview-datahub/organization-preview-datahub.component.html

```diff
@@ -0,0 +1,23 @@
+<div class="w-72">
+  <div
+    class="border p-3 rounded-lg my-2 w-full h-48 flex justify-center items-center my-3"
+  >
+    <img [src]="logo" alt="title" />
+  </div>
+  <div class="px-3 capitalize">
+    <span class="font-bold my-2 text-lg"> {{ title }}</span>
+    <p class="text-gray-800 my-2">{{ description }}</p>
```

Comment:
I don't see the clamp thing ?
What happens if the text is very long ? Should be cropped